scripts/install-chef-suse.sh

```diff
@@ -71,6 +78,8 @@ if [ -n "$CROWBAR_FROM_GIT" ]; then
             install -p -m 0755 $tool /opt/dell/bin/
         fi
     done
+else
+    BARCLAMP_INSTALL_OPTS="$BARCLAMP_INSTALL_OPTS --rpm"
```

Comment:
Sorry, I didn't mean "lose in the patch", but we had it in the past and it seems we don't have it right now (until before your patch)?